.github/workflows/ci.yml

```
@@ -38,7 +46,7 @@ jobs:
       - run: |
           python -m pip install --upgrade pip wheel
           pip install --upgrade -r requirements_test.txt
-          pip install --upgrade --progress-bar off https://github.com/pyvista/pyvista/zipball/main https://github.com/spyder-ide/qtpy/zipball/master
+          pip install --upgrade --progress-bar off https://github.com/larsoner/pyvista/zipball/pyvistaqt https://github.com/spyder-ide/qtpy/zipball/master
```

Before we merge, need to revert this
```suggestion
          pip install --upgrade --progress-bar off https://github.com/pyvista/pyvista/zipball/main https://github.com/spyder-ide/qtpy/zipball/master
```